src/pages/Transaction.vue

```diff
@@ -63,6 +64,9 @@ export default {
         await this.loadTransaction();
     },
     methods: {
+        formatWei(number, decimals){
```

Comment:
since the signature is the same here, it could be simplified to 

```js
methods: {
    formatWei,
    resetTransaction() {
        ...
    }
}

```